launcher/ui/WinDarkmode.h

```diff
@@ -0,0 +1,70 @@
+#ifndef WINDARKMODE_H
+#define WINDARKMODE_H
+
+#include <QWidget>
+
+#ifdef Q_OS_WIN
+#include <Windows.h>
```

Comment:
this is that time where i say something i know 0% about, but i've seen around the code base we using `#include <windows.h>` (with the minuscule w), like in https://github.com/PolyMC/PolyMC/blob/0e473f4570f49ba212459365c7971b5d3988a3ff/launcher/FileSystem.cpp#L48. i don't know if there's any difference, but i guess we should use one or the other (?)

Reply:
The difference is that Windows.h is wrong and windows.h is right, but it only really cares when cross compiling

Reply:
Yes, I will change that.